content/product-engineering/planning.md

```diff
@@ -0,0 +1,214 @@
+| Week 10 of current quarter<br>2 weeks until next quarter | Finalize OKRs<ul> <li>VPP/VPE & Product/Eng Directors align on final OKRs during [VP team sync](team.md#vp-team-sync)</li> <li>VPP/VPE get exec team buy in to finalize and approve OKRs</li> <li>OKRs added to the handbook & communicated on Slack</li> </ul> |
+| Week 12 of current quarter<br>1 week until next quarter | End of quarter retrospective<ul> <li>VPP/VPE & Product/Eng Directors retro previous quarter</li> <li>VPP/VPE and CTO team retro previous quarter & finalize OKR scorecard</li> </ul> |
+| Week 1 of new quarter | Kick off new quarter<ul> <li>VPP/VPE share previous quarter OKR scorecard to Product/Eng Department </li> <li>Communicate department OKRs for upcoming quarter at Company kick off</li> </ul> |
+
+## Sharing progress
+
+There are a few ways we track plans and celebrate progress at Sourcegraph.
+
+- The [GitHub project tracker](#github-project-tracker) is **the source of truth** for all key results in product and engineering.
+- [OKR slides](#okr-slides) contain the top level product and engineering results that the VPs and Directors review and discuss on a weekly basis.
+- [Monthly team updates](#monthly-team-updates) communicate a high-level narrative/summary to the rest of the company about what each team in product/eng accomplished since the last monthly update and what is up next.
+- [Monthly demo days](demo-day.md) are a great opportunity for engineers to highlight and share with a broad audience the specific things they shipped. It is really fun!
+
+### Status definitions
+
+| Status        | Description                                  |
+| ------------- | -------------------------------------------- |
+| 🟢 On track   | 90-100% confidence we will achieve the goal. |
+| 😬 At risk    | 75-89% confidence we will achieve the goal.  |
+| ⛔️ Off track | <75% confidence we will achieve the goal.    |
+| 🚀 Done       | We have achieved the goal!                   |
+| ☠️ Cancelled  | We are no longer pursuing this goal.         |
+
+### GitHub project tracker
+
+The [GitHub project tracker](https://github.com/orgs/sourcegraph/projects/214) is **the source of truth** for key results in product and engineering. There are a few tabs:
+
+- The **Product and Engineering** tab tracks the results that the product and engineering VPs and directors review every week at the [VP team sync](#vp-team-sync).
+- Each org (Cloud, Enablement, Code graph) has a tab that tracks the results for that org. Org directors review this dashboard each week with their team.
+- The **Engineering** tab tracks results that are relevant only to engineering or engineering leadership. This is reviewed every week in the [VPE team sync](#todo-link).
```

Comment:
Leftover TODO link here

@jyavorska  shouldn't the link checker have caught this? 🤔

Reply:
@felixfbecker I removed the link checker I made because it didn't work properly. The only way to really make it work consistently is post-build and export.